netboot-oci-artifacts.md

```diff
@@ -0,0 +1,96 @@
+# Packaging Network Boot files
+
+Netboot files (i.e. PXE) can be packaged as [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
+using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+and stored in OCI registry.
+
```

Comment:
All required files MUST be stored in the registry together in order to either boot into container, or boot into installer which is responsible for deploying bootable image. The registry MUST NOT contain any associated configuration files required for booting.